lib/perl/Genome/Qc/Tool/Picard/MarkDuplicates.t

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env genome-perl
+
+BEGIN { 
+    $ENV{UR_DBI_NO_COMMIT} = 1;
+    $ENV{UR_USE_DUMMY_AUTOGENERATED_IDS} = 1;
+}
+
+use strict;
+use warnings;
+
+use above "Genome";
+use Test::More;
+use Sub::Override;
+
+my $pkg = 'Genome::Qc::Tool::Picard::MarkDuplicates';
+use_ok($pkg);
+
+my $data_dir = __FILE__.".d";
+
+my $metrics_file = File::Spec->join($data_dir, 'output_file.txt');
+
+my $tool = $pkg->create(
+    gmt_params => {
+        input_file => __FILE__,
+        metrics_file => $metrics_file,
+        output_file => __FILE__,
```

Comment:
This makes me a little bit nervous - I know we're not executing the command, but is there a chance that someone would add that to the test and then it could stomp over the source file?